external/CMakeLists.txt

```diff
@@ -59,4 +58,40 @@ if (${ENABLE_SPIRV_CODEGEN})
       message(FATAL_ERROR "GoogleTest was not found - required for SPIR-V codegen")
     endif()
   endif()
-endif()
\ No newline at end of file
+
+  # We need Effcee for SPIR-V codegen testing.
+  if (${SPIRV_BUILD_TESTS})
+    set(DXC_RE2_DIR "${DXC_EXTERNAL_ROOT_DIR}/re2" CACHE STRING "Location of RE2 source")
```

Comment:
Maybe we should put an explanation here to say that `re2 is needed since it's required by effcee.`